src/nav.json

```diff
@@ -170,6 +170,10 @@
             {
               "name": "Managing template versions",
               "slug": "template-versions"
+            },
```

Comment:
It feels like plugins and custom domains should be next to each other in the sidebar. Maybe move the plugins below template versions. What do you think?